tests/codegen/waiter-tests/src/test/kotlin/com/test/FunctionTypeTest.kt

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.test
+
+import com.test.model.*
+import com.test.utils.successTest
+import com.test.waiters.*
+import org.junit.jupiter.api.Test
+
+class FunctionTypeTest {
+    @Test fun testTypeFunctionStringEquals() = successTest(
```

Comment:
style: consider `@Test` on a separate line

Reply:
I think it makes more sense to have `@Test` like that for now. Only because the rest of the [tests](https://github.com/awslabs/smithy-kotlin/tree/3934d48a9e084e23b1250501b0deb8dafe2a1c03/tests/codegen/waiter-tests/src/test/kotlin/com/test) use the same style. I can refactor all to use that style just in another PR to keep things easy to review

Reply:
Ok, I agree with keeping it consistent